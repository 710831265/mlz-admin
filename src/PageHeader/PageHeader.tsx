import React, { useState, useContext } from 'react';
import { PageHeader as AntdPageHeader } from 'antd';

const PageHeader = AntdPageHeader;

export default PageHeader;
