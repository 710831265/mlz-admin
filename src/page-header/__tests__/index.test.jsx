import React, { Component } from 'react';
import testMount from '../../../tests/testMount';
import PageHeader from '..';

describe('ğŸ“¦  PageHeader', () => {
  testMount(PageHeader);
});
