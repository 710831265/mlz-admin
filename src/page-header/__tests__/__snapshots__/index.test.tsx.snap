// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ðŸ§ª PageHeader snapshot test snapshot comparing 1`] = `"<div class=\\"ant-page-header ant-page-header-ghost\\"></div>"`;
